HVZ/HVZ/api/views.py

```diff
@@ -61,7 +71,7 @@ def form_valid(self, form):
         elif(recipient_title == MailerForm.ZOMBIES):
             recipients = [p.user.email for p in Player.current_players() if p.team == "Z"]        
 
-        # TODO: Authentication error for sender for mod@claremonthvz.org
+        # send mail based on the given inputs
         send_mail(subject, body, sender, recipients)
```

Comment:
An alert of failed to send to an email address will be turned on once we have the basic functionalities.